spec/requests/api/providers_spec.rb

```diff
@@ -686,4 +686,58 @@ def failed_auth_action(id)
       expect_results_to_match_hash("results", [failed_auth_action(p1.id), failed_auth_action(p2.id)])
     end
   end
+
+  describe 'Providers import VM' do
+    let(:provider)      { FactoryGirl.create(:ems_redhat, sample_rhevm) }
+    let(:provider_url)  { providers_url(provider.id) }
+
+    let(:vm)            { FactoryGirl.create(:vm_vmware) }
+    let(:vm_url)        { vms_url(vm.id) }
+
+    let(:cluster)       { FactoryGirl.create(:ems_cluster) }
+    let(:cluster_url)   { clusters_url(cluster.id) }
+
+    let(:storage)       { FactoryGirl.create(:storage) }
+    let(:storage_url)   { data_stores_url(storage.id) }
+
+    NAME = 'new_vm_name'.freeze
+
+    it 'rejects import without appropriate role' do
+      api_basic_authorize
+
+      run_post(providers_url(100), gen_request(:import_vm))
```

Comment:
If you are testing authorization, can you keep all the other factors the same as the happy path test?